mmcls/datasets/transforms/formatting.py

```diff
@@ -156,12 +156,10 @@ def transform(self, results: dict) -> dict:
                 'please make sure `LoadImageFromFile` has been added '
                 'in the data pipeline or images have been loaded in ')
 
-        data_sample = MultiTaskDataSample(tasks=self.tasks)
-        gt_label = {}
-        for task in self.tasks:
-            if task in results:
-                gt_label[task] = to_tensor(results[task])
-        data_sample.set_gt_label(gt_label)
+        data_sample = MultiTaskDataSample(metainfo=self.tasks)
```

Comment:
Let's call it `self.metainfo`
```suggestion
        data_sample = MultiTaskDataSample(metainfo=self.metainfo)
```